yarn.lock

```diff
@@ -6616,11 +6616,6 @@ hds-core@3.10.0:
   resolved "https://registry.yarnpkg.com/hds-core/-/hds-core-3.10.0.tgz#19956a8b8600611d8b3188b8d5305229f11eeb2b"
   integrity sha512-fF2Ub5fd+NMgoOwYv0k1LmaWsjnJr4hif1vXPp0Y9VbRGtHAN5/MsR0a1zb1NW+ygA0/5V5+ti8kXFutoKu7XA==
 
-hds-design-tokens@^3.7.0:
-  version "3.7.0"
-  resolved "https://registry.yarnpkg.com/hds-design-tokens/-/hds-design-tokens-3.7.0.tgz#233636beeff5df39d47cfb4e4861e760a085c3c3"
-  integrity sha512-JBfVmsDQbLAPwX1Xh0uX5o3q4xfqQSQSWx4vewXmOBTjNkGcf2au+XCo9S5QvETai9OR85m5GbTh33DXquFLXw==
-
 hds-design-tokens@^3.9.0:
   version "3.10.0"
```

Comment:
Please note that hds-core, hds-react and hds-design-tokens all use 3.10.0 in this pull request, they're installed as ^3.9.0 which resolves to 3.10.0. Looking at RHHC newest master they are using 3.9.0 for hds-core, hds-react and hds-design-tokens. I'm just making sure you are aware of this, this may be ok if this works. So, I'd say let's test!

Reply:
i tested, it should work